### ISSUE TITLE
Run of docker container comdb2-dev fails due to missing folder

### ISSUE THREAD

**Issue by dennisgove** [bloomberg/comdb2#64]
When running the container built with `make docker-dev` it fails to create a database because directory `/var/cdb2` does not exist.

**Comment by mponomar** [comment]
They should end up in `/opt/bb/var/cdb2'.  Prefix is missing.  Wonder if broken #54.

**Comment by mponomar** [comment]
Broken by #61.  Which also breaks package builds.

**Comment by mponomar** [comment]
Fixed by #66.  Give that a shot.

```
mike@xps:~/comdb2/src$ docker run --name test -d comdb2-dev:latest db1 db2 db3
e9e5e0b22696c01be4cbb4c45421db0a5b9e5fdd9ca22e37e91eec0a1b469471
mike@xps:~/comdb2/src$ docker exec -ti test /bin/cdb2config > config
mike@xps:~/comdb2/src$ export CDB2_CONFIG_FILE=config
mike@xps:~/comdb2/src$ for db in db1 db2 db3; do cdb2sql $db default 'select comdb2_dbname()'; done
(comdb2_dbname()='db1')
(comdb2_dbname()='db2')
(comdb2_dbname()='db3')
mike@xps:~/comdb2/src$ docker kill test
test
mike@xps:~/comdb2/src$ for db in db1 db2 db3; do cdb2sql $db default 'select comdb2_dbname()'; done
cdb2_open rc -1 cdb2_get_dbhosts: can't do dbinfo query on db1 hosts.
cdb2_open rc -1 cdb2_get_dbhosts: can't do dbinfo query on db2 hosts.
cdb2_open rc -1 cdb2_get_dbhosts: can't do dbinfo query on db3 hosts.

```

**Comment by mponomar** [comment]
This is fix - was breaking due to a broken build.
